drivers/eeprom/eeprom_fm24w256.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2021 Caspar Friedrich <c.s.w.friedrich@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <devicetree.h>
+#include <devicetree/gpio.h>
+#include <drivers/eeprom.h>
+#include <drivers/gpio.h>
+#include <drivers/i2c.h>
+#include <logging/log.h>
+
+#define DT_DRV_COMPAT cypress_fm24w256
+
+#define LEVEL POST_KERNEL
+#define PRIORITY CONFIG_I2C_INIT_PRIORITY
```

Comment:
Please just use these constants directly in the code. Having a level of indirection here does not improve code readability/understanding.

Reply:
Done